conda-recipe/meta.yaml

```diff
@@ -51,8 +51,8 @@ requirements:
         - sphinxcontrib-bibtex
         - curio >=1.0
         - pineappl >=0.5.8
-        - eko ==0.10.2
-        - banana-hep ==0.6.6
+        - eko >=0.12.0
+        - banana-hep >=0.6.8
         - lz4 >=3.1.10 # see https://github.com/conda-forge/eko-feedstock/issues/10
```

Comment:
This issue is actually solved, I believe here you now want greater equal than `4.0` (`4.0.2` to be aligned with that version of EKO)

Reply:
[e00d11d](https://github.com/NNPDF/nnpdf/pull/1694/commits/e00d11d53d05c7c42cdcc5afd0d98a7d4f29a714)